src/how-to-do-links-to-other-pages.wiki

```diff
@@ -1,28 +1,63 @@
 =How to do links to other pages?=
 
-Examples:
+Sometimes, you need to create links to other websites. For Eliom, these links are called «external services». These external services are services which are not registered and generated by your application (for example a link to https://ocsigen.org).
+
+You can create an external service with
+<<code language="ocaml"|
+let ocsigen_service =
+  Eliom_service.create
+    ~id:(Eliom_service.External ("https://ocsigen.org", [""]))
+    ~meth:(Eliom_service.Get Eliom_parameter.unit)
+    ()
+>>
+
+This code create an external service for https://ocsigen.org. The function Eliom_service.External takes a tuple (a, b) where a is the URL to the page as a string and the second is an array representing the path. Another example for the page https://ocsigen.org/tuto/manual:
+<<code language="ocaml"|
+let ocsigen_tuto_manual_service =
+  Eliom_service.create
+    ~id:(Eliom_service.External ("https://ocsigen.org", ["tuto", "manual"]))
+    ~meth:(Eliom_service.Get Eliom_parameter.unit)
+    ()
+>>
+
+By changing the parameter ~meth, you can also send GET and POST parameters, like for an internal service (ie registered by your application). For example, if http://example.com needs three GET parameters of type int, int and string, you create an external service to the site with
```

Comment:
That's an explanation of how services work. As such, it is beyond the point in a howto about links (which can have no POST parameters).